server/src/main/java/org/elasticsearch/indices/recovery/PeerRecoveryTargetService.java

```diff
@@ -189,51 +192,11 @@ private void doRecovery(final long recoveryId) {
                 return;
             }
         }
-
-        try {
-            logger.trace("{} starting recovery from {}", request.shardId(), request.sourceNode());
-            final AtomicReference<RecoveryResponse> responseHolder = new AtomicReference<>();
-            cancellableThreads.execute(() -> responseHolder.set(
```

Comment:
The [declaration and assignment of `cancellableThreads` above](https://github.com/elastic/elasticsearch/pull/37076/files#diff-cdd186e2c33a5fa4d4200f9119684443L177) is no longer used.

Reply:
++ thanks